packages/salesforcedx-apex-debugger/src/messages/index.ts

```diff
@@ -21,17 +23,15 @@ function loadMessageBundle(config?: Config): Message {
       : `${BASE_FILE_NAME}.${locale}.${BASE_FILE_EXTENSION}`;
   }
 
-  const base = new Message(
-    require(`./${resolveFileName(DEFAULT_LOCALE)}`).messages
-  );
+  const base = new Message(messages);
 
   if (config && config.locale && config.locale !== DEFAULT_LOCALE) {
     try {
-      const layer = new Message(
-        require(`./${resolveFileName(config.locale)}`).messages,
-        base
-      );
-      return layer;
+      if (config.locale === 'ja') {
```

Comment:
OK disclaimer: I haven't ran this code yet.
Instead of throwing an exception, since this is a know situation, how about the following instead:
```
.
.
.
  const base = new Message(
    require(`./${resolveFileName(DEFAULT_LOCALE)}`).messages
  );

  if (config && config.locale && config.locale !== DEFAULT_LOCALE) {
    if (config.locale === 'ja') {
      const layer = new Message(jaMessages, base);
      return layer;
    }
  }


  return base;
}
```

Reply:
I'm fine with that.  Will update both places